### PR TITLE
Docker setup

### PR DESCRIPTION
**Pull request by netochaves**
#27 Added docker and updated README.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+FROM ruby:2.6.3
+
+ENV NODE_VERSION=8.12.0
```
Comment (marclerodrigues) [inline Dockerfile]: Is this the latest stable version of Node?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+run sed -i '/deb-src/d' /etc/apt/sources.list && \
+  apt-get update
+
+run apt-get install -y build-essential postgresql-client
+run gem install bundler
```
Comment (marclerodrigues) [inline Dockerfile]: This is not the best approach, we need to use volumes so after we ran `docker-compose down` we don't need to run bundle install again,

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+RUN curl -sSL "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-x64.tar.xz" | tar xfJ - -C /usr/local --strip-components=1 && \
+  npm install npm -g
+
+run npm install --global yarn
+
+RUN mkdir /myapp
+WORKDIR /myapp
+COPY Gemfile /myapp/Gemfile
+COPY Gemfile.lock /myapp/Gemfile.lock
+RUN bundle install
```
Comment (marclerodrigues) [inline Dockerfile]: This is not the best approach, we need to use volumes so after we ran docker-compose down we don't need to run bundle install again,

File: config/database.yml.sample [hunk 4]
```diff
@@ -19,7 +19,10 @@ default: &default
   encoding: unicode
   # For details on connection pooling, see Rails configuration guide
   # https://guides.rubyonrails.org/configuring.html#database-pooling
-  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  host: db
```
Comment (marclerodrigues) [inline config/database.yml.sample]: I would create a `config/database.yml.docker-sample` so users that prefer not using can copy this file instead.

**Comment by marclerodrigues** [comment]
@netochaves take a look into https://github.com/marclerodrigues/headhunter42/blob/master/Dockerfile
https://github.com/marclerodrigues/headhunter42/blob/master/docker-compose.yml

File: Dockerfile (hunk 1 above)
Comment (netochaves) [reply Dockerfile]: Actually not. I'll update

File: Dockerfile (hunk 3 above)
Comment (netochaves) [reply Dockerfile]: ok

File: config/database.yml.sample (hunk 4 above)
Comment (netochaves) [reply config/database.yml.sample]: Yeah, sounds good

**Pull request merged**